src/smallMultiples.js

```diff
@@ -68,7 +68,7 @@ export default function smallMultiples(id, chart) {
         rangePolygon(this);
 
         // axis tweaks
-        var units = this.current_data[0].values.raw[0][chart.config.unit_col];
+        var units = this.raw_data[0][chart.config.unit_col];
```

Comment:
For the moment, maybe we should create/maintain a `chart.measure_data` in this renderer. Seems like that would deal with most of this nonsense in a fairly elegant way ... just throw something like this in to `onInit()` and `onDraw()`:

```
var current_measure = {go get it from the control}
chart.measure_data = raw_data.filter(d=>d[chart.config.measure_col]==current_measure)
```